pkg/common/googlegroup_github/mapper.go

```diff
@@ -71,31 +68,19 @@ type BiDirectionalGroupMapper struct {
 	TargetMapper *GroupMapper
 }
 
-// NewBidirectionalGoogleGroupGitHubMapper creates a GoogleGroupToGitHubMapper
-// and a GitHubToGoogleGroupMapper using the provided groupMapping file.
-// Returns is (GoogleGroupToGitHubMapper, GitHubToGoogleGroupMapper, error).
-//
-// TODO: refactor this into client/googlegroup_github/mapper.go later.
-func NewBidirectionaGroupMapper(groupMappingFile string) (*BiDirectionalGroupMapper, error) {
-	b, err := os.ReadFile(groupMappingFile)
-	if err != nil {
-		return nil, fmt.Errorf("failed to read mapping file: %w", err)
-	}
-	var tm v1alpha3.GoogleGroupToGitHubTeamMappings
-	if err := prototext.Unmarshal(b, &tm); err != nil {
-		return nil, fmt.Errorf("failed to unmarshal mapping file: %w", err)
-	}
+func NewBidirectionaGroupMapper(mappings *api.GroupMappings) *BiDirectionalGroupMapper {
```

Comment:
This part is basically the same as previous, the only difference is removed the textproto parsing because it's handled in the caller functions.